### DISCUSSION TITLE
A way for analysts without sensor creds to pull Strelka files?

### DISCUSSION THREAD

**Discussion by samanosuke26** [Security-Onion-Solutions/securityonion#9263] [Unsupported Versions] [answered]
Security Onion 2.3.130

With our current setup, our sensor administrators pull alerted strelka files for the analysts using WinSCP or similar. 

Is there a current or built-in way for the analysts to grab those files themselves through the SOC or Kibana?

**Comment by dougburks** [comment, marked as answer]
One option would be to pivot to full packet capture and then extract the file:
https://docs.securityonion.net/en/2.3/pcap.html

**Reply by samanosuke26** [reply]
Thank you Doug, I managed to extract the file from cyberchef and compared the hash with the alert ensuring it was the same file. Drafting up an SOP for our analysts now!
